### PR TITLE
feat: remove last updated alert from homepage

### LINKED ISSUE #598: Remove the "last updated" alert on the home page
Let's remove the "last updated" alert on the home page.
[I have gotten the confirmation](https://kawalcovid19.slack.com/archives/C029KS7SWSJ/p1628473595000100?thread_ts=1628417748.001400&cid=C029KS7SWSJ) that we don't need the "last updated" alert anymore.

Screenshot 2021-08-09 at 10 17 58

_Originally posted by @zainfathoni in https://github.com/kawalcovid19/wargabantuwarga.com/issues/321#issuecomment-894912512_

## Acceptance Criteria

- [x] Remove the component rendering from the home page
- [x] Remove the widget from Netlify CMS
- [x] ~Update the failing tests accordingly~

### PR DESCRIPTION
**Pull request by antoooks**
Closes #598 

## Description

Remove "last updated" alert from homepage

## Current Tasks

- [x] Remove the component rendering from the home page
- [x] Remove the widget from Netlify CMS
- [x] Update the failing tests accordingly

### CODE REVIEW

File: __tests__/pages/index.test.tsx [hunk 1]
```diff
@@ -11,11 +11,11 @@ describe("HomePage", () => {
...
-        /Pembaruan terakhir pada Selasa, 27 Juli 2021 17.43 WIB/i,
-      ),
-    ).toBeVisible();
+    const LastUpdatedAlert = screen.queryByText(
+      /Pembaruan terakhir pada Selasa, 27 Juli 2021 17.43 WIB/i,
+    );
+
+    expect(LastUpdatedAlert).not.toBeInTheDocument();
   });
 });
```
Comment (pandutiket) [inline __tests__/pages/index.test.tsx:20]: I think we can remove this test, wdyt @zainfathoni @antoooks?

Comment (pandubaraja) [inline __tests__/pages/index.test.tsx:20]: I think we can remove this test, wdyt @zainfathoni @antoooks?

Comment (antoooks) [reply __tests__/pages/index.test.tsx]: Good point, actually this is a question from me as well. If we remove a component altogether from existing page, should we test the if the component not exist or delete the test altogether?

Comment (antoooks) [reply __tests__/pages/index.test.tsx]: Removed, I think we can resolve this conversation

**Review by mazipan — APPROVED** [review]
Looks good to me

But the codecov still failed.

Help to fix it first, Mostly on `components/ui/alert/**`

File: __tests__/pages/index.test.tsx [hunk 2]
```diff
@@ -1,24 +1,8 @@
...
-    render(<HomePage latestNews={[latestNewsItemBuilder()]} />);
-
-    expect(
-      screen.getByText(
-        /Pembaruan terakhir pada Selasa, 27 Juli 2021 17.43 WIB/i,
-      ),
-    ).toBeVisible();
-  });
-});
-
```
Comment (zainfathoni) [inline __tests__/pages/index.test.tsx:21]: Thanks for working on it, @antoooks.
Hiwever, this test removal reduced the code coverage of the home page. Please put it back and change the assertion instead.

FYI, Mas @mazipan & @resir014.

**Review by zainfathoni — COMMENTED** [review]
We shouldn't decrease the test coverage of any [well-covered code](https://app.codecov.io/gh/kawalcovid19/wargabantuwarga.com/compare/604/diff#diff-cGFnZXMvaW5kZXgudHN4). 😅

I'm reopening #598 issue to restore the test properly.

File: components/ui/__tests__/alert.test.tsx [hunk 3]
```diff
@@ -45,5 +52,12 @@ describe("Alert", () => {
       "border-red-400",
     );
     expect(container.firstChild?.firstChild).toHaveClass("p-4", "babayaga");
+    expect(
+      container.firstChild?.firstChild?.firstChild?.firstChild?.firstChild,
```
Comment (zainfathoni) [inline components/ui/__tests__/alert.test.tsx:56]: Please [query the component that you're targeting properly](https://testing-library.com/docs/queries/about).

File: components/ui/__tests__/alert.test.tsx [hunk 4]
```diff
@@ -45,5 +52,12 @@ describe("Alert", () => {
...
+    ).toHaveClass("flex-shrink-0 mr-3");
+    expect(
+      container.firstChild?.firstChild?.firstChild?.childNodes[1],
```
Comment (zainfathoni) [inline components/ui/__tests__/alert.test.tsx:59]: Please [query the component that you're targeting properly](https://testing-library.com/docs/queries/about).

File: components/home/__tests__/homepage-content.test.tsx [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+import { render } from "@testing-library/react";
+import { HomePageContent } from "../homepage-content";
+
+describe("homepage content render correctly", () => {
+  const { container } = render(<HomePageContent />);
+
+  it("renders correctly", () => {
+    expect(container.firstChild).toHaveClass("flex-1");
```
Comment (zainfathoni) [inline components/home/__tests__/homepage-content.test.tsx:10]: Please perform more meaningful assertions that's obvious to the user experience other than merely asserting this `flex-1` class.

**Review by zainfathoni — COMMENTED** [review]
I have more requests to fix the way we're testing the components here.

**Pull request merged**